docs/UserGuide.md

```diff
@@ -359,6 +528,20 @@ Format: `exit`
 **A**: Copy over the data folder (containing friendsList.json and gamesList.json) generated by gitGud on your current computer into the folder containing the 
 gitGud.jar on the other computer.
 
+**Q**: Which operating systems do gitGud support? <br>
+**A**: gitGud is developed and tested to support most popular Operating systems including Microsoft Windows 10, Mac OS and
+on Linux. However, as there are many different distributions of Linux, we are not able to test and guarantee full support 
+for all distributions. 
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Glossary:
```

Comment:
Thank you for adding this in.
Can perhaps consider adding in friendId and gameId meanings here as well.